temperature/utils.py

```diff
@@ -0,0 +1,27 @@
+import json
+
+import httpx
+from fastapi import HTTPException
+
+from settings import settings
+
+
+WEATHER_URL = (
+    f"http://api.weatherapi.com/v1/current.json?key={settings.WEATHER_TOKEN}"
+)
+
+
+async def get_temperature(city_name: str) -> float:
+    async with httpx.AsyncClient() as client:
+        response = await client.get(WEATHER_URL + f"&q={city_name}")
```

Comment:
The URL concatenation here directly appends the city name to the `WEATHER_URL`. Ensure that the city name is properly encoded to handle spaces or special characters in city names.